internal/syncer/connector/box_test.go

```
@@ -0,0 +1,645 @@
+		return boxFile{}, nil
+	}
+	connector.downloadFile = func(ctx context.Context, fileID string, sizeThreshold int64) ([]byte, error) {
+		downloadIDs = append(downloadIDs, fileID)
+		return []byte("body:" + fileID), nil
+	}
+
+	session, err := connector.OpenSync(context.Background(), SyncRequest{FromBeginning: true})
+	if err != nil {
+		t.Fatalf("OpenSync failed: %v", err)
+	}
+	defer session.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `session.Close` is not checked

(errcheck)